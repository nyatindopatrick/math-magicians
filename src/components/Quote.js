import React from 'react';

export default function Quote() {
  return (
    <div className="main">
      <p>
        Mathematics is not about numbers, equations, computations or algorithms:
        it is about understanding. - William Paul Thurston
      </p>
    </div>
  );
}
