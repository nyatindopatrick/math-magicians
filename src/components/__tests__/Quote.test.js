import React from 'react';
import rendere from 'react-test-renderer';

import Quote from '../Quote';

describe('Quote', () => {
  it('Includes text "equations"', () => {
    const tree = rendere.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
