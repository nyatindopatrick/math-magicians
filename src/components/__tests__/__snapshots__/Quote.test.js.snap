// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Includes text "equations" 1`] = `
<div
  className="main"
>
  <p>
    Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. - William Paul Thurston
  </p>
</div>
`;
